app/src/main/java/io/apicurio/registry/rest/RulesResourceImpl.java

```diff
@@ -55,12 +65,19 @@
     @Current
     RegistryStorage storage;
 
+    @Inject
+    RulesProperties rulesProperties;
+
     /**
      * @see io.apicurio.registry.rest.RulesResource#listGlobalRules()
      */
     @Override
     public List<RuleType> listGlobalRules() {
-        return storage.getGlobalRules();
+        List<RuleType> rules = storage.getGlobalRules();
+        List<RuleType> defaultRules = rulesProperties.getDefaultGlobalRules(rules);
+        return Stream.concat(rules.stream(), defaultRules.stream())
+            .sorted()
+            .collect(Collectors.toList());
```

Comment:
If the same rule is configured in both places, won't it show up twice in the list?

Reply:
`getDefaultGlobalRules()` takes a list of RuleType enums that will be filtered out of the returned list. So this concatenation of the 2 lists will only include the global rules that are in storage plus any default global rules that aren't in the list from storage.

I have renamed `getDefaultGlobalRules()` to `getFilteredDefaultGlobalRules()` in the latest commit to hopefully make it clearer that it's (potentially) a sub-list that is returned.

Reply:
Ah yes - that **is** the part I missed.  I didn't notice that it was passing in the rules.